driver/normalizer/parser.go

```diff
@@ -12,9 +12,8 @@ var ToNoder = &native.ObjectToNoder{
 	TokenKeys:          map[string]bool{"text": true},
 }
 
-// ASTParserBuilder creates a parser that transform source code files
-// into *uast.Node.
-func UASTParserBuilder(opts driver.UASTParserOptions) (driver.UASTParser, error) {
+// Creates a parser that transform source code files into *uast.Node.
+func ParserBuilder(opts driver.ParserOptions) (driver.Parser, error) {
```

Comment:
Comments for public symbols in Go must start with the symbol name (`ParserBuilder creates a parser...`)... or that's what Vim plugins always tells me.